crates/ruff/src/rules/ruff/rules/asyncio_dangling_task.rs

```diff
@@ -55,7 +56,7 @@ define_violation!(
 impl Violation for AsyncioDanglingTask {
     #[derive_message_formats]
     fn message(&self) -> String {
-        format!("Store a reference to the return value of `asyncio.create_task`")
+        format!("Store a reference to the return value of `asyncio.create_task` and `asyncio.ensure_future`")
```

Comment:
Let's tailor this message based on the function call. If you take a look at the `DeferralKeyword` enum in `crates/ruff/src/rules/pyflakes/rules/yield_outside_function.rs`, we can do the same thing here: add it as a member on the rule struct, then include it in the format message.

Reply:
Change is made! :smile: